client-side/src/Routes.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Route, Routes, Router } from 'react-router-dom';
+import MyThoughts from './pages/MyThoughts';
+import Favourites from './pages/Favourites';
+import AddMyThought from './pages/AddMyThought';
+import AboutApp from './pages/AboutApp';
+import Home from './pages/Home';
+
+const AppRoutes: React.FC = () => {
+  return (
+        <Routes>
+        <Route path="/" element={<Home />}/>
```

Comment:
Indent this portion for better visual appeal in your code.

Reply:
Sure thing! Thank you